patches/minecraft/net/minecraft/world/gen/feature/jigsaw/JigsawPatternRegistry.java.patch

```diff
@@ -0,0 +1,12 @@
+--- a/net/minecraft/world/gen/feature/jigsaw/JigsawPatternRegistry.java
++++ b/net/minecraft/world/gen/feature/jigsaw/JigsawPatternRegistry.java
+@@ -11,6 +11,9 @@
+       this.func_214932_a(JigsawPattern.field_214949_a);
+    }
+ 
++   /**
++    * use {@link net.minecraftforge.event.world.jigsaw.JigsawStructurePoolInitEvent#registerNewJigsawPattern(JigsawPattern)} for registering new patterns for existing structures
```

Comment:
Do not add javadoc to vanilla methods

Reply:
Use the bot instead

Reply:
References to forge code/patches should definitely not be in MCP comments...